package.json

```diff
@@ -45,6 +45,9 @@
     "testVM": "node ./tests/tester -v",
     "testState": "node ./tests/tester -s",
     "testBlockchain": "node --stack-size=1500 ./tests/tester -b",
+    "testBlockchainBlockGasLimit": "node --stack-size=1500 ./tests/tester -b --dir='bcBlockGasLimitTest'",
+    "testBlockchainValid": "node --stack-size=1500 ./tests/tester -b --dir='bcValidBlockTest'",
+    "testBlockchainTotalDifficulty": "node --stack-size=1500 ./tests/tester -b --dir='bcTotalDifficultyTest'",
```

Comment:
Ideally, these could be one line:
```
"testBlockchainTravis": "node --stack-size=1500 ./tests/tester -b --dir='bcBlockGasLimitTest testBlockchainValidtest BlockchainTotalDifficulty'
```

Reply:
1) ``--dir`` really is a directory option - directly passed through to ``ethereumjs-testing``, where you can select e.g. all the ``GeneralStateTests`` in ``BlockChainTests`` with ``--dir='GeneralStateTests'`` or a subfolder (test cases) with ``--dir='GeneralStateTests/stCallCodes'``. For just renaming ``--excludeDir`` this doesn't seems worth the effort, since this would have to be routed through the already merged PR on ``ethereumjs-testing``.

2) Since including several directories isn't supported in the current version of ``ethereumjs-testing`` this would need a wrapper in the ``ethereumjs-vm`` tests implementation and I think this is more confusing than helpful for people reading the code and expecting the same API as in ``ethereumjs-testing``. So I also would say this isn't worth the effort here, this doesn't have to scale or something, it's just a practical solution to get three test folders runnning.

Reply:
I actually thought longer than it might seem on how to implement the ``--dir`` option (e.g. not using the filters from ``npm-dir``. The selected solution has some flexibility in some directions (e.g. the subfolder selection from above wouldn't be possible with the current filter implementation in ``npm-dir``, and it is generally easier to use (with ``npm-dir`` filter I actually experimented over half an hour and didn't get the results I expected).

So the ``--dir`` implementation lacks a filter functionality but I think the use case for this for directories is relatively limited, though one can probably come up with use cases (all directories with ``memory`` in it, hmm :-)). If this should be needed in the future an option ``--dirFilter`` could additionally be added.

It is also possible to expand the current ``--dir`` implementation with the possibility to add several directories in the future and not loosing backwards compatibility. 

To sum up: for our current use cases I decided that this is the better choice. :-)